<ui version="4.0">
<class>MainWindow</class>
<widget class="QMainWindow" name="MainWindow" >
<property name="geometry" >
<rect>
<x>0</x>
<y>0</y>
<width>400</width>
<height>300</height>
</rect>
</property>
<property name="windowTitle" >
<string>RobotStore</string>
</property>
<widget class="QMenuBar" name="menuBar" />
<widget class="QToolBar" name="mainToolBar" />
<widget class="QWidget" name="centralWidget">
<widget class="QWidget" name="layoutWidget">
<property name="geometry">
<rect>
<x>40</x>
<y>30</y>
<width>501</width>
<height>441</height>
</rect>
</property>
<layout class="QGridLayout" name="gridLayout">
<item row="0" column="0">
<layout class="QHBoxLayout" name="horizontalLayout">
<item>
<widget class="QLabel" name="label">
<property name="text">
<string>PLC IP : </string>
</property>
</widget>
</item>
<item>
<widget class="QLineEdit" name="plcIP"/>
</item>
<item>
<widget class="QLabel" name="label_2">
<property name="text">
<string>Port : </string>
</property>
</widget>
</item>
<item>
<widget class="QLineEdit" name="plcPort"/>
</item>
<item>
<widget class="QPushButton" name="connectBtn">
<property name="text">
<string>Connect</string>
</property>
</widget>
</item>
</layout>
</item>
<item row="1" column="0">
<layout class="QHBoxLayout" name="horizontalLayout_2">
<item>
<widget class="QLabel" name="label_4">
<property name="text">
<string>Server IP : </string>
</property>
</widget>
</item>
<item>
<widget class="QLineEdit" name="serverIP"/>
</item>
<item>
<widget class="QLabel" name="label_3">
<property name="text">
<string>Port : </string>
</property>
</widget>
</item>
<item>
<widget class="QLineEdit" name="serverPort"/>
</item>
<item>
<widget class="QPushButton" name="listenBtn">
<property name="text">
<string>Listen</string>
</property>
</widget>
</item>
</layout>
</item>
<item row="2" column="0">
<widget class="QTextBrowser" name="tcpMessage"/>
</item>
<item row="3" column="0">
<widget class="QPushButton" name="uiStartBtn">
<property name="text">
<string>Start UI</string>
</property>
</widget>
</item>
</layout>
</widget>
<widget class="QCheckBox" name="cbSingleTest">
<property name="geometry">
<rect>
<x>560</x>
<y>30</y>
<width>83</width>
<height>16</height>
</rect>
</property>
<property name="text">
<string>SingleTest</string>
</property>
</widget>
</widget>
<widget class="QMenuBar" name="menuBar">
<property name="geometry">
<rect>
<x>0</x>
<y>0</y>
<width>1246</width>
<height>21</height>
</rect>
</property>
</widget>
<widget class="QToolBar" name="mainToolBar">
<attribute name="toolBarArea">
<enum>TopToolBarArea</enum>
</attribute>
<attribute name="toolBarBreak">
<bool>false</bool>
</attribute>
</widget>
<widget class="QStatusBar" name="statusBar" />
</widget>
<layoutDefault spacing="6" margin="11" />
<pixmapfunction></pixmapfunction>
<resources/>
<connections/>
</ui>
